configure.ac

```diff
@@ -114,6 +117,9 @@ PKG_CHECK_MODULES([DEPENDENCIES], [
 		  gcr-3 >= $GCR_REQUIRED
 		  avahi-gobject >= $AVAHI_REQUIRED
 		  avahi-client >= $AVAHI_REQUIRED
+		  json-glib-1.0 >= $JSON_GLIB_REQUIRED
```

Comment:
You'll have a conflict here too, since I split this out into many separate calls to PKG_CHECK_MODULES. When you rebase, you'll need to adjust the preprocessor and linker flags for libephymain_la as needed, or you won't be able to build. I can help with this, and you don't need to bother with it until you rebase.